tests/test_source_lt_from_config.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from nzshm_model.logic_tree import InversionSource, SourceLogicTree
+
+
+def test_slt_from_config():
```

Comment:
@chrisdicaprio suggest some edge case tests here - showing errors thrown for invalid setup ??